index.ts

```diff
@@ -30,84 +37,85 @@ import { Blob as NodeBlob } from 'buffer';
 import { authorizationTypeEnum as authType } from './AuthorizationType.js';
 import 'isomorphic-fetch';
 import { isBrowser } from '@laserfiche/lf-js-utils/dist/utils/core-utils.js';
+import * as fsPromise from 'fs/promises';
 
 let _RepositoryApiClient: IRepositoryApiClient;
 const rootFolderEntryId = 1;
-const sampleProjectEdocName = 'JS Sample Project GetDocumentContent';
+const sampleProjectDocumentName = 'JS Sample Project GetDocumentContent';
+const largeDocumentFilePath = 'testFiles/sample.pdf';
 
 await main();
 
 async function main(): Promise<void> {
   try {
-    const scope = 'repository.Read,repository.Write';
+    const scope = 'repository.Read repository.Write';
     if (authorizationType === authType.CloudAccessKey) {
       _RepositoryApiClient = createCloudRepositoryApiClient(scope);
     } else {
       _RepositoryApiClient = createSelfHostedRepositoryApiClient();
     }
-    await getRepositoryName(); //Print repository name
-    await getFolder(rootFolderEntryId); //Print root folder name
-    await getFolderChildren(rootFolderEntryId); //Print root folder children
-    const createFolderEntry = await createFolder(); //Creates a sample project folder
-    const tempEdocEntryId = await importDocument(createFolderEntry.id, sampleProjectEdocName); //Imports a document inside the sample project folder
-    await setEntryFields(createFolderEntry.id); // Set Entry Fields
-    await getFolder(createFolderEntry.id); //Print sample project folder name
-    await getFolderChildren(createFolderEntry.id); //Print sample project folder children
-    await getEntryFields(createFolderEntry.id); // Print entry Fields
-    await getEntryContentType(tempEdocEntryId); // Print Edoc Information
-    await searchForImportedDocument(sampleProjectEdocName); //Search for the imported document inside the sample project folder
-    await deleteSampleProjectFolder(createFolderEntry.id); // Deletes sample project folder and its contents inside it
+    await printAllRepositoryNames();
+    await printFolderName(rootFolderEntryId);
+    await printFolderChildrenInformation(rootFolderEntryId);
+    const sampleFolderEntry = await createSampleProjectFolder();
+    const importedEntryId = await importDocument(sampleFolderEntry.id, sampleProjectDocumentName);
+    await setEntryFields(importedEntryId);
+    await printEntryFields(importedEntryId);
+    await searchForImportedDocument(sampleProjectDocumentName);
+    await importLargeDocument(sampleFolderEntry.id, largeDocumentFilePath);
+    await deleteSampleProjectFolder(sampleFolderEntry.id);
```

Comment:
Do you think it is a good idea to try to delete the sample project folder even if there was an error thrown? Otherwise the sample won't clean up itself

Reply:
Fixed.